<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>572</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="inputTextEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="submitRequestButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>50</y>
      <width>91</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Submit Request</string>
    </property>
   </widget>
   <widget class="QComboBox" name="requestTypeComboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>82</width>
      <height>28</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>GET</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>POST</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>DELETE</string>
     </property>
    </item>
   </widget>
   <widget class="QTextBrowser" name="responseTextBrowser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>551</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>551</width>
      <height>221</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab_none">
     <attribute name="title">
      <string>none</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_raw">
     <attribute name="title">
      <string>JSON</string>
     </attribute>
     <widget class="QTextEdit" name="bodyTextEdit">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>-1</y>
        <width>549</width>
        <height>193</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>572</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menudd">
    <property name="title">
     <string>sharcAPI client</string>
    </property>
   </widget>
   <addaction name="menudd"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
